### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by charlielizzy**
All but one cypress test now working

### CODE REVIEW

File: src/Components/RecordingButton/index.jsx [hunk 1]
```diff
@@ -1,41 +1,56 @@
...
+                  props.setAlbum(album);
+                  props.setArtist(artist);
+                  props.setTitle(title);
+                  props.setArtwork(artwork)   
+                  // props.setSpotifyID(spotifyID)             
+                }
               })
-              .catch((err) => ('Error occurred: we could not locate a song', err))
+              .catch((err) => {console.log('Error occurred: API fail', err); return (
+                <h1 data-automation="apiCall">API fail</h1>
```
Comment (TomGroombridge) [inline src/Components/RecordingButton/index.jsx]: you can't do this. you need to set some state or redirect the user or something, you can't add html to javascript code outside the template

File: src/Components/RecordingButton/index.jsx [hunk 2]
```diff
@@ -1,41 +1,56 @@
...
+              )})
             }).catch((e) => {
-              alert('We could not record your song');
+              alert('Recordimg failed');
```
Comment (TomGroombridge) [inline src/Components/RecordingButton/index.jsx]: spelling

Comment (charlielizzy) [reply src/Components/RecordingButton/index.jsx]: fixed

File: cypress/e2e/auddio-API-tests.cy.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
+describe('happy-path', () => {
+  describe('intial setup of app when loaded', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000')
```
Comment (TomGroombridge) [inline cypress/e2e/auddio-API-tests.cy.js:4]: i think we could add cy.clock here as even though it's not used with all tests it should not affect them

File: cypress/e2e/auddio-API-tests.cy.js [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    })
+    it('record button should be reenabled after 10 seconds of being disabled', () => {
+      cy.clock()
+      cy.get('[data-automation="record-button"]').click()
+      cy.get('[data-automation="record-button"]').should('be.disabled')
+      cy.tick(11000)
+      cy.get('[data-automation="record-button"]').should('not.be.disabled')
+    })
+
+    it('should return songNotFound if result is equal to null and menuBar should not be visible', () => {
```
Comment (TomGroombridge) [inline cypress/e2e/auddio-API-tests.cy.js]: This test could live in it's own describe as it's specific to the scenario where song not found.

File: src/Components/RecordingButton/index.jsx (hunk 1 above)
Comment (charlielizzy) [reply src/Components/RecordingButton/index.jsx]: set as state

File: cypress/e2e/auddio-API-tests.cy.js (hunk 3 above)
Comment (charlielizzy) [reply cypress/e2e/auddio-API-tests.cy.js:4]: done

File: cypress/e2e/auddio-API-tests.cy.js (hunk 4 above)
Comment (charlielizzy) [reply cypress/e2e/auddio-API-tests.cy.js]: added song not found describe

**Review by TomGroombridge — APPROVED** [review]

**Pull request merged**